app/src/main/java/org/xbmc/kore/ui/sections/file/MediaFileListFragment.java

```diff
@@ -323,7 +324,7 @@ private void browseDirectory(final FileLocation dir) {
 
         Files.GetDirectory action = new Files.GetDirectory(dir.file,
                 mediaType,
-                new ListType.Sort(ListType.Sort.SORT_METHOD_LABEL, true, true),
+                new ListType.Sort(ListType.Sort.SORT_METHOD_PATH, true, true),
```

Comment:
I will need some time to implement this, so it would be nice if you could merge this change as is. I'll try to look into this though - if I succeed I'll send another PR.
I'm still not completely happy with SORT_METHOD_PATH anyway, as some files don't seem to get sorted by path/filename, but I can't figure out why.